single_tech_samples/databricks/sample1_basic_azure_databricks_environment/README.md

```diff
@@ -0,0 +1,210 @@
+
+Tools like Azure Resource Manager (ARM), Terraform, and the Azure Command Line Interface (CLI) enable you to declaratively script the cloud infrastructure and use software engineering practices such as testing and versioning while implementing IaC.
+
+This sample will focus on automating the provisioning of a basic Azure Databricks environment using the Infrastructure as Code pattern
+
+### 1.1. Scope
+
+The following list captures the scope of this sample:
+
+1. Provision an Azure Databricks environment using ARM templates orchestrated by a shell script.
+1. The following services will be provisioned as a part of the basic Azure Databricks environment setup:
+   1. Azure Databricks Workspace
+   2. Azure Storage account with hierarchical namespace enabled to support ABFS
+   3. Azure key vault to store secrets and access tokens
+
+Details about [how to use this sample](#3-how-to-use-this-sample) can be found in the later sections of this document.
+
+### 1.2. Architecture
+
+The below diagram illustrates the deployment process flow followed in this sample:
+
+![alt text](../Common_Assets/Images/IAC_Architecture.png "Logo Title Text 1")
+
+#### 1.2.1. Patterns
+
+Following are the cloud design patterns being used by this sample:
+
+- [External Configuration Store pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/external-configuration-store): Configuration for the deployment is persisted externally as a parameter file separate from the deployment script.
+- [Federated Identity pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/federated-identity): Azure active directory is used as the federated identity store to enable seamless integration with enterprise identity providers.
+- [Valet Key pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/valet-key): Azure key vault is used to manage the secrets and access toked used by the services.
+- [Compensating Transaction pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/compensating-transaction#): The script will roll back partially configured resources if the deployment is incomplete.
+
+### 1.3. Technologies used
+
+The following technologies are used to build this sample:
+
+- [Azure Databricks](https://azure.microsoft.com/en-au/free/databricks/)
+- [Azure Storage](https://azure.microsoft.com/en-au/services/storage/data-lake-storage/)
+- [Azure Key Vault](https://azure.microsoft.com/en-au/services/key-vault/)
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/)
+- [Azure Resource Manager](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/overview)
+
+## 2. Well-Architected Framework (WAF)
```

Comment:
To reduce redundancy -- I think we can make the WAF section more concise across the samples. For sample 1, I think we can safely remove it since its meant to be starter example.

Reply:
@namitms Addressing

Reply:
@devlace Resolved